Gemfile.lock

```diff
@@ -8,8 +8,8 @@ GEM
     em-websocket (0.5.2)
```

Comment:
If you could reset this file as well that would be great! Since this PR doesn't add any new dependencies or revolve around fixing the Gemfile(.lock) we shouldn't be pushing it. We should open a separate PR for that.